back/apps/core-fca/src/controllers/oidc-client.controller.ts

```diff
@@ -275,6 +276,21 @@ export class OidcClientController {
     @UserSessionDecorator(GetOidcCallbackSessionDto)
     userSession: ISessionService<UserSession>,
   ) {
+    if (req.query['error']) {
+      const dict = {};
+      dict[req.query['error']] = req.query['error_description'];
+      res.status(400);
+      return res.render('error', {
+        exception: {},
+        error: {
+          code: req.query['error'],
+          id: uuid(),
```

Comment:
As far as I know, the point of generating unique IDs for errors is to allow support personnel to look up more details (e.g. stack traces) in a log or event tracking system when a user reports an error they encountered.

Generating a random ID that is not otherwise logged anywhere feels quite pointless unless I missed something.